.changeset/tame-keys-provide.md

```diff
@@ -0,0 +1,18 @@
+---
+'@primer/react-brand': minor
+---
+
+Use VideoPlayer component to render hosted videos
+
+```jsx
+<VideoPlayer title="HLS On Demand Demo" crossOrigin="anonymous">
+  <VideoPlayer.Source src="https://cdn.api.video/vod/vi36GGvd6PoTqViQLxBWwHjJ/mp4/1080/source.mp4" />
+  <VideoPlayer.Track
+    src="https://raw.githubusercontent.com/JoshBowdenConcepts/Subtitles-Example/main/sample.vtt"
+    default
+    kind="subtitles"
+    srcLang="en"
+    label="English"
+  />
```

Comment:
I think only `kind` and `src` are required for `<track>`. Could we either remove these other ones or make them required props on VideoPlayer.Track if they are necessary?

Reply:
This has been updated in the following [SHA](https://github.com/primer/brand/pull/373/commits/05ec5df1318a6dfeb932bbc4b8e240cf0fd0e3bf)

Reply:
Kind was not required so I removed that as well.